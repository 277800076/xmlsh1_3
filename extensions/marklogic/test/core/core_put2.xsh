# 
# Test -collection
#

. ../init
ml:put -uri /test1.xml -collection foo -collection bar <[ <foo>bar</foo> ]>
ml:put -uri /test2.xml -collection foo <[ <foo>bar</foo> ]>
ml:put -uri /test3.xml -collection bar <[ <foo>bar</foo> ]>
ml:put -uri /test4.xml <[ <foo>bar</foo> ]>


echo foo,bar
ml:query 'for $u in fn:collection(("foo","bar"))/base-uri() order by $u return $u'
echo foo
ml:query 'for $u in  fn:collection("foo")/base-uri() order by $u return $u'
echo bar
ml:query 'for $u in fn:collection("bar")/base-uri()  order by $u return $u'
echo none
ml:query 'fn:collection("none")/base-uri()'





ml:deldir /



