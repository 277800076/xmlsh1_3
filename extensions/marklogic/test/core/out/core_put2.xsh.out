foo,bar
/test1.xml
/test2.xml
/test3.xml
foo
/test1.xml
/test2.xml
bar
/test1.xml
/test3.xml
none
